cmd/hello/FyneApp.toml

```diff
@@ -4,3 +4,6 @@
   ID = "io.fyne.hello"
   Version = "2.5.3"
   Build = 2
+
+[Migrations]
+  fyneDo = true
```

Comment:
Same comment as further down. Maybe calling this `threading = true` would make more sense?

Reply:
Hmm, true but I'm not all too enthusiastic about the previous name either. I would like to hear what @dweymouth thinks.

Reply:
If we are to do a v3.0.0, then we can remove the tag and use the same tag again in the future in case we want to do one more thread migration (which I kind of think is unlikely before v3.0.0). I am still in favour of migrating threads as a better name (`fynedo` sounds a bit strange) but I approved this for now so it can land and be useful for more people.